apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx

```diff
@@ -284,6 +284,7 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
                         {isInput && !isSelect && (
                           <FormControl>
                             <Input
+                              {...field}
```

Comment:
`field` had a `disabled` property that was set to `undefined`. This was reproduced only when the backoffice was installed as a standalone project (not part of a monorepo)